portfolio/src/main/java/com/google/sps/data/CommentListSingleton.java

```diff
@@ -0,0 +1,53 @@
+package com.google.sps.data;
+
+import java.util.List;
+import java.util.ArrayList;
+import java.util.Map;
+import java.util.HashMap;
+
+import com.google.sps.dataExceptions.CommentExistingId;
+import com.google.sps.dataExceptions.CommentNotFound;
+
+public class CommentListSingleton {
```

Comment:
Same as PR 13. Please modify it to store and retrieve comments for different projects separately.

Reply:
This comment has to be resolved though.

Reply:
As discussed, I moved the comment section on a separate page and added a link on the bottom of the page. If I have time remaining on week 4 or 5, I will add a link to the /comments.html page on the menu!